### DISCUSSION TITLE
Possible to add USB/Wired controller support?

### DISCUSSION THREAD

**Discussion by nikolasjunqueira** [darthcloud/BlueRetro#303] [Ideas]
It is possible to add wired/usb controller support to use with the consoles? like ps2 usb on xbox.

**Comment by darthcloud** [comment]
While nothing is stopping anybody from daisy-chaining adapter, it's not something I will officially support.

And I won't add native USB support to BlueRetro for the reason already given in #84

**Reply by nikolasjunqueira** [reply]
thanks for reply, but my question is the inverse of #84. i mean use usb controller or diy esp32 bluetooth controller to play in the consoles, like dreamcast etc. is possible to connect 2 esp32 to each other? one as transmisor ant other as a Blueretro

**Reply by darthcloud** [reply]
The answer is the same, no and for the same reason.

**Reply by darthcloud** [reply]
Like if you make a Bluetooth controller with ESP32 it will work.

But its not something I got time to do myself.
